app/services/account_reset/grant_requests_and_send_emails.rb

```diff
@@ -2,9 +2,10 @@ module AccountReset
   class GrantRequestsAndSendEmails
     def call
       notifications_sent = 0
+      tvalue = Time.zone.now - Identity::Hostdata.settings.account_reset_wait_period_days.to_i.days
       AccountResetRequest.where(
         sql_query_for_users_eligible_to_delete_their_accounts,
-        tvalue: Time.zone.now - AppConfig.env.account_reset_wait_period_days.to_i.days,
+        tvalue: tvalie,
```

Comment:
typo

Reply:
Thanks!

Also just realized not going ahead with this for now .. going to close it